src/cljs/rems/new_application.cljs

```diff
@@ -19,7 +19,7 @@
             {:params {:catalogue-item-ids catalogue-item-ids}
              :handler (fn [response]
                         (remove-catalogue-items-from-cart! catalogue-item-ids)
-                        (application/navigate-to (:application-id response) true))
+                        (replace-url! (str "/application/" (:application-id response))))
```

Comment:
why not `navigate!`?

Reply:
It's to remove the current URL (`/application?items=1`) from the browser history. Otherwise pressing the browser's back button will create a new application.

Reply:
The old method used to take a boolean parameter for switching this behaviour, but I split it into two separate methods (`navigate!` and `replace-url!`).